plugins/bazaar/api-report.md

```diff
@@ -38,6 +38,12 @@ export const bazaarPlugin: BackstagePlugin<
 // @public (undocumented)
 export const EntityBazaarInfoCard: () => JSX.Element | null;
 
+// @public (undocumented)
+export type HomePageProps = {
```

Comment:
Could we name this `BazaarPageProps` since that matches the public interface rather than the internal one?

You could keep its name unchanged internally but _only exporting_ it with the new name if you want to.